contracts/Web3Task.sol

```diff
@@ -65,7 +72,7 @@ abstract contract Web3Task is ERC721, AccessControl, IWeb3Task {
         }
 
         uint256 balance = _balances[_task.creatorRole];
-        if (_task.reward > balance) {
+        if (_task.reward > balance || _task.reward < 10e12) {
```

Comment:
I'm wondering why the fixed 10e12 in this conditional, is really necessary. What is it doing since we already have it validated in the first section of the conditional when checking the balance? I think we can remove this addition, what do you think @RafaDSan ?

Reply:
We did another review and this seems non-necessary, could you remove this validation then commit so we can merge this PR @RafaDSan ?

Reply:
@0xneves In fact the contracts folder should not be in this repo. I'll make an issue to remove the unnecessary files from this repository!